docs/sphinx/using/simulators.rst

```diff
@@ -80,38 +80,98 @@ simulator that will be used.
 Tensor Network Simulators
 ==================================
 
-cuQuantum multi-node multi-GPU
-++++++++++++++++++++++++++++++++++
+CUDA Quantum provides a couple of tensor-network simulator targets accelerated with 
+the :code:`cuTensorNet` library. 
+These backends are available for use from both C++ and Python.
 
-The :code:`tensornet` target provides a tensor-network simulator accelerated with 
-the :code:`cuTensorNet` library. This backend is currently available for use from C++ and supports 
-Multi-Node, Multi-GPU distribution of tensor operations required to evaluate and simulate the circuit.
+`cuTensorNet` multi-node multi-GPU
```

Comment:
Capitalization of multi-node multi-GPU is inconsistent with line 92. I would recommend making them consistent or capitalizing the one in line 87 since it is a subject heading and uncapitalizing the one in line 92. My GitHub suggestions are for the latter, but I'm ok with either way.
```suggestion
`cuTensorNet` Multi-Node Multi-GPU
```